### PR TITLE
Allow pjm file to be used instead pf pj2 file

### PR DESCRIPTION
**Pull request by dkeza**
We are using [PushOK](http://www.pushok.com/software/svn.html) extension to use Subversion 1.5.9 in VFP9 IDE for version control.
Because of that, we have PJM files for VFP PJX files in Subversion repository.
foxbin2prg.cfg is set as this:
```
extension: pj2=pjm
extension: vc2=vca
extension: sc2=sca
extension: fr2=fra
extension: lb2=lba
extension: mn2=mna
extension: db2=dba
extension: dc2=dca
```
When we now will to create text files from binary files
```
foxbin2prg d:\keza\code\develop\vfp\test2\testproject.pjx "*"
```
this works.
But, when we try to generate PJX file from pjm file:
```
foxbin2prg d:\keza\code\develop\vfp\test2\testproject.pjm "*"
```
this doesn't work, because of special treatment of PJM file in foxbin2prg.prg.

Because of this, we changed foxbin2prg.prg so that in case when instead of pj2 is used pjm file, don't do anything special with PJM files.

### CODE REVIEW

**Comment by fdbozzo** [comment]
I dkeza!

Very thanks for reporting this problem with pjm files.
In this case the special treatment of pjm files is because it is mainly used to allow the recreation of the PJX from the SourceSafe created pjm file, which have a special format.

In this case I can't use your proposed code because this will disable the SourceSafe-pjm compatibility format, creating a regression, BUT I can add a rule for which if you reassign the pj2 extension as pjm (as you need in this case, using pj2=pjm), then do not use the SourceSafe compatibility and create a normal pj2 file with pjm extension.

In short I let you know about the availability of the modification.

Stay tuned! :)

**Comment by fdbozzo** [comment]
Hi Darko!

Good news: it's fixed in v1.19.50.1

Let me know if this bugfix is working for you too.
(I added this as issue: [https://github.com/fdbozzo/foxbin2prg/issues/19](https://github.com/fdbozzo/foxbin2prg/issues/19))

Thanks!

**Comment by fdbozzo** [comment]
Handled as bugfix and fixed in [Issue#19](https://github.com/fdbozzo/foxbin2prg/issues/19)
